icubam/backoffice/server.py

```diff
@@ -0,0 +1,45 @@
+import os.path
+
+import tornado.ioloop
+import tornado.locale
+import tornado.web
+from absl import logging
+from tornado_sqlalchemy import SQLAlchemy
```

Comment:
this might indeed be interesting to have an async driver. but for now i would rather just use our DB abstraction, that now uses SQLAlchemy. So you can replace this line by `from tornado.db import sqlite`